.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
You shouldn't include these .idea files in your PR. You should:
* Remove these files from the .idea
* add `.idea` to .gitignore in the base of the repo